src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +8,19 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, Toa Payoh Lorong 8, #03-10, 123456";
```

Comment:
good that you have updated the help in the program. additionally you need to update:
1. user guide
2. tests